book_source/source/module-5-classes-and-objects/lesson-14-more-classes-and-modules/private-fields.md

```diff
@@ -5,6 +5,7 @@
     <iframe src="https://www.loom.com/embed/de5c457d29854cad9093650e160fc9da?sharedAppSource=personal_library" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe>
 </div>
 
+---
 
 ```{admonition} Tip
```

Comment:
As another comment for a line of code I can't directly comment on:

Maybe we should add a note admonition at the top of the `EdPost` practice problem saying we use EdStem as a discussion board which is why we call this problem `EdPost`? Or we could change this problem to be `DiscussionPost` instead but that seems like more work haha